src/style.css

```
@@ -0,0 +1,94 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: italic;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-1 {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 0 62px;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

```suggestion
  font-family: Avenir, sans-serif;
```
Move it to the `html` tag selector